Library/Homebrew/cmd/test-bot.rb

```diff
@@ -480,6 +480,7 @@ def formula formula_name
       test "brew", "install", "--build-bottle", *changed_dependences unless changed_dependences.empty?
       formula_fetch_options = []
       formula_fetch_options << "--build-bottle" unless ARGV.include? "--no-bottle"
+      formula_fetch_options << "--build-from-source" unless ARGV.include? "--build-bottle"
```

Comment:
Oh, also, this behaviour explicitly isn't what we want given https://github.com/DomT4/homebrew/blob/soapy_water/Library/Homebrew/cmd/test-bot.rb#L705-L707